src/backend/routes/feature-toggles.ts

```diff
@@ -22,3 +23,19 @@ export const konfigurerFeatureTogglesEndpoint = (app: Express): Express => {
     app.get(`${basePath}toggles/:id(familie-ba-soknad.[a-zA-Z-]{0,})`, toggleFetchHandler);
     return app;
 };
+
+const fetchAllFeatureTogglesHandler: RequestHandler<
+    ParamsDictionary,
+    Ressurs<EAllFeatureToggles>
+> = (_, res) => {
+    res.send(
+        byggSuksessRessurs({
+            EØS_KOMPLETT: isEnabled(ToggleKeys.EØS_KOMPLETT),
+        })
+    );
+};
+
+export const konfigurerAllFeatureTogglesEndpoint = (app: Express): Express => {
+    app.get(`${basePath}toggles/all`, fetchAllFeatureTogglesHandler);
```

Comment:
Hvorfor trenger vi alle togglene? Ser at i konfigurerFeatureTogglesEndpoint på linje 20 i filen så henter man bare de togglene som ligger innenfor "familie-ba-soknad" burde man legge til dette på basepathen til denne også så vi bare får de togglene som "tilhører" oss? 🤔

Reply:
Tror dette tyder på litt uheldig navngivning fra min side 😅  Fra unleash henter vi kun 1 toggle, som er EØS_KOMPLETT. Det blir gjort på linje 33 (isEnabled funksjonen). 

Tanken med dette endepunktet er at vi henter alle toggles som vi bryr oss om, for å slippe å gjøre ett kall per toggle fra frontend.